features/guidance/bugs.feature

```diff
@@ -0,0 +1,35 @@
+@routing @guidance
+Feature: Features related to bugs
+
+    Background:
+        Given the profile "car"
+        Given a grid size of 5 meters
+
+    @2852
+    Scenario: Loop
+        Given the node map
+            | a | 1 |   | g |   |   | b |
+            |   |   |   |   |   |   |   |
+            |   |   |   |   |   |   |   |
+            | e |   |   |   |   |   | f |
+            |   |   |   |   |   |   |   |
+            |   |   |   |   |   |   | 2 |
+            | d |   |   | h |   |   | c |
+
+        And the ways
+            | nodes | name   | oneway |
+            | agb   | top    | yes    |
+            | bfc   | right  | yes    |
+            | chd   | bottom | yes    |
+            | dea   | left   | yes    |
+
+        And the nodes
+            | node | highway         |
+            | g    | traffic_signals |
+            | f    | traffic_signals |
+            | h    | traffic_signals |
+            | e    | traffic_signals |
```

Comment:
Well. It should work just the same without. The test here covers more, though, since we check both identical names not collapsed and simple name changes in between. The general issue here is that we need a disconnected loop of some form which will never reach anything else but a two way intersection.